src/entity/ProjectFactory.java

```diff
@@ -0,0 +1,17 @@
+package entity;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public interface ProjectFactory {
+    /**
+     * @param projectTitle - the title of the project
+     * @param tasks - an ArrayList representing the subtasks that the project has been broken down to, to put into the
+     * calendar. Each task is represented by a HashMap, mapping an attribute of the task to its value. Each subtask will
+     * have the following keys and values:
+     *      - TaskName: <name of the task>
+     *      - TaskDescription: <description of the task>
+     *      - TaskDeadline: <deadline of the task>
+     */
+    Project create(String projectTitle, ArrayList<HashMap<String, String>> tasks);
```

Comment:
ISSUE: change this to work with Task entity

Reply:
This is in regards to a mock entity that was never used. It was committed by accident, I will remove it.